qiskit/circuit/quantumcircuit.py

```diff
@@ -1249,28 +1225,16 @@ def _append(
 
         Args:
             instruction: Instruction instance to append
-            qargs: qubits to attach instruction to
-            cargs: clbits to attach instruction to
+            qargs: Qubits to attach the instruction to.
+            cargs: Clbits to attach the instruction to.
 
         Returns:
             Instruction: a handle to the instruction that was just added
 
-        Raises:
-            CircuitError: if the gate is of a different shape than the wires
-                it is being attached to.
+        :meta public:
```

Comment:
Ah it's good to be publishing this now especially with the appropriate warnings

Reply:
This is the right Sphinx incantation to publish it, but I think our `autosummary` templates actually suppress it right now anyway.  Thanks for reminding me to take a look - either way, it'll be a follow-up PR.